templates/worker-kv-shortener/src/index.ts

```diff
@@ -0,0 +1,39 @@
+import { logger } from "hono/logger";
+import { Hono } from "hono";
+import { cors } from "hono/cors";
+
+export interface Env {
+	SHORTENER_KV: KVNamespace;
+	HOST_URL: string;
+}
```

Comment:
This would encourage people to handcraft their types, I would like to see either comment mentioning `wrangler types` or steps in the "getting started" in README.